src/components/styles/Divider.tsx

```diff
@@ -1,9 +1,14 @@
 import styled from "@emotion/styled";
-import { Divider } from "antd";
+import { palette } from "@leafygreen-ui/palette";
 import { size } from "constants/tokens";
 
-const StyledDivider = styled(Divider)`
+const { gray } = palette;
+
+const Divider = styled("hr")`
+  background-color: ${gray.light2};
+  border: 0;
+  height: 1px;
   margin: ${size.xxs} 0 ${size.xs} 0;
```

Comment:
My preference for this component would probably be to have the same margin on top and bottom